pkg/vault/contracts/test/VaultAdminMock.sol

```diff
@@ -55,13 +60,33 @@ contract VaultAdminMock is IVaultAdminMock, VaultAdmin {
         IVault(address(this)).initializeBuffer(wrappedToken, amountUnderlying, amountWrapped, sharesOwner);
     }
 
+    function manualAddLiquidityToBuffer(IERC4626 wrappedToken, uint256 underlyingAmount, uint256 wrappedAmount) public {
+        bytes32 bufferBalances = _bufferTokenBalances[wrappedToken];
+
+        if (underlyingAmount > 0) {
+            IERC20(wrappedToken.asset()).transferFrom(msg.sender, address(this), underlyingAmount);
+            _reservesOf[IERC20(wrappedToken.asset())] += underlyingAmount;
+            _bufferTotalShares[wrappedToken] += underlyingAmount;
+        }
+        if (wrappedAmount > 0) {
+            IERC20(address(wrappedToken)).transferFrom(msg.sender, address(this), wrappedAmount);
+            _reservesOf[IERC20(address(wrappedToken))] += wrappedAmount;
+            _bufferTotalShares[wrappedToken] += wrappedToken.previewRedeem(wrappedAmount);
+        }
```

Comment:
we're not minting shares to the caller; can't we just mint the assets to the vault instead of using `transferFrom`?
Otherwise consider assigning shares to the caller.

Reply:
Since this was just a test, I did in the quicker way posible, but I'll send the shares to the caller.